api/core/services/users/clerk_user_service.py

```diff
@@ -72,6 +73,26 @@ async def get_org_admins(self, org_id: str, max_users: int = 5) -> list[UserDeta
                 return []
             return await self._get_users_by_id(client, user_ids)
 
+    async def get_user_id_by_email(self, email: str) -> str:
```

Comment:
Can we cache the result ? Clerk rates limit are very small

Reply:
Addressed in https://github.com/WorkflowAI/WorkflowAI/pull/176/commits/63e63a9ee9b79e05667a321b73e4b8ff7632dd33

This is a cache update because we want to ignore the self / cls arguments.